tuxemon/core/components/save.py

```diff
@@ -58,15 +60,43 @@ def save(player, slot, game):
     >>> core.components.save.save(player1, 2, self)
 
     """
-
+    path = '.saves/'
+    try:
+            os.makedirs(path)
+    except OSError:
+            if not os.path.isdir(path):
+                        raise
     # Save a screenshot of the current frame
-    pygame.image.save(game.save_screenshot, 'saves/slot' + str(slot) + '.png')
-
-    saveFile = shelve.open('saves/slot' + str(slot) + '.save')
+    pygame.image.save(game.save_screenshot, '.saves/slot' + str(slot) + '.png')
+    saveFile = shelve.open('.saves/slot' + str(slot) + '.save')
     saveFile['game_variables'] = game.player1.game_variables
     saveFile['tile_pos'] = game.player1.tile_pos
-    saveFile['inventory'] = game.player1.inventory
-    saveFile['current_map'] = game.event_engine.current_map.filename
+    tempinv1 = {}
+    tempinv1 = game.player1.inventory
+    for keysinv, valuesinv in tempinv1.items():
+        for keys2inv, values2inv in valuesinv.items():
+            if keys2inv == 'item':
+                values2inv.surface = None
+    saveFile['inventory'] = tempinv1
+    saveFile['current_map'] = game.event_engine.current_map.filename.split("/")[-1]
+    tempmon1 = []
```

Comment:
This should be `tempmon1 = list(game.player1.monsters)` or something similar. The current way you have it here will actually clear the sprites from the `game.player1.monsters` list.

Here's an example of what you're doing right now. Anything you do to `mylist_2` will affect `mylist_1`:
```python
>>> mylist_1 = [1, 2, 3]
>>> mylist_2 = mylist_1
>>> mylist_2.pop()
3
>>> mylist_2
[1, 2]
>>> mylist_1
[1, 2]
```

Doing `mylist_2 = list(mylist_1)` will have this effect:
```python
>>> mylist_1 = [1, 2, 3]
>>> mylist_2 = list(mylist_1)
>>> mylist_2.pop()
3
>>> mylist_2
[1, 2]
>>> mylist_1
[1, 2, 3]
```